<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>958</width>
    <height>561</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label_trem1">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>30</y>
      <width>21</width>
      <height>17</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel { background: blue}</string>
    </property>
    <property name="text">
     <string>T1</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_trilho1">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>30</y>
      <width>250</width>
      <height>17</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel { background: yellow}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_trilho2">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>150</y>
      <width>250</width>
      <height>17</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel { background: yellow}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_trilho3">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>30</y>
      <width>21</width>
      <height>137</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel { background: yellow}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_trilho4">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>30</y>
      <width>21</width>
      <height>137</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel { background: yellow}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>410</y>
      <width>99</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Ligar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>410</y>
      <width>98</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Parar</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_trilho4_2">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>30</y>
      <width>21</width>
      <height>137</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel { background: yellow}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_trilho1_2">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>30</y>
      <width>250</width>
      <height>17</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel { background: yellow}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_trilho2_2">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>150</y>
      <width>250</width>
      <height>17</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel { background: yellow}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_trem2">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>30</y>
      <width>21</width>
      <height>17</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel { background: red}</string>
    </property>
    <property name="text">
     <string>T2</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_trilho2_3">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>150</y>
      <width>250</width>
      <height>17</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel { background: yellow}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_trilho4_3">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>30</y>
      <width>21</width>
      <height>137</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel { background: yellow}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_trilho1_3">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>30</y>
      <width>250</width>
      <height>17</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel { background: yellow}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_trem3">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>30</y>
      <width>21</width>
      <height>17</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel { background: green}</string>
    </property>
    <property name="text">
     <string>T3</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_trilho2_4">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>270</y>
      <width>250</width>
      <height>17</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel { background: yellow}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_trilho4_4">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>150</y>
      <width>21</width>
      <height>137</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel { background: yellow}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_trilho4_5">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>150</y>
      <width>21</width>
      <height>137</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel { background: yellow}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_trilho2_5">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>270</y>
      <width>250</width>
      <height>17</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel { background: yellow}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_trilho3_2">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>150</y>
      <width>21</width>
      <height>137</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel { background: yellow}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_trem5">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>150</y>
      <width>21</width>
      <height>17</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel { background: orange}</string>
    </property>
    <property name="text">
     <string>T5</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_trem4">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>150</y>
      <width>21</width>
      <height>17</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel { background: purple}</string>
    </property>
    <property name="text">
     <string>T4</string>
    </property>
   </widget>
   <widget class="QLabel" name="critical_1">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>80</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QLabel" name="critical_2">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>80</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QLabel" name="critical_3">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>150</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>3</string>
    </property>
   </widget>
   <widget class="QLabel" name="critical_4">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>150</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>4</string>
    </property>
   </widget>
   <widget class="QLabel" name="critical_5">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>150</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>5</string>
    </property>
   </widget>
   <widget class="QLabel" name="critical_6">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>150</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>6</string>
    </property>
   </widget>
   <widget class="QLabel" name="critical_7">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>210</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>7</string>
    </property>
   </widget>
   <zorder>label_trilho1</zorder>
   <zorder>label_trilho2</zorder>
   <zorder>label_trilho3</zorder>
   <zorder>label_trilho4</zorder>
   <zorder>label_trem1</zorder>
   <zorder>pushButton</zorder>
   <zorder>pushButton_2</zorder>
   <zorder>label_trilho4_2</zorder>
   <zorder>label_trilho1_2</zorder>
   <zorder>label_trilho2_2</zorder>
   <zorder>label_trem2</zorder>
   <zorder>label_trilho2_3</zorder>
   <zorder>label_trilho4_3</zorder>
   <zorder>label_trilho1_3</zorder>
   <zorder>label_trem3</zorder>
   <zorder>label_trilho2_4</zorder>
   <zorder>label_trilho4_4</zorder>
   <zorder>label_trilho4_5</zorder>
   <zorder>label_trilho2_5</zorder>
   <zorder>label_trilho3_2</zorder>
   <zorder>label_trem5</zorder>
   <zorder>label_trem4</zorder>
   <zorder>critical_1</zorder>
   <zorder>critical_2</zorder>
   <zorder>critical_3</zorder>
   <zorder>critical_4</zorder>
   <zorder>critical_5</zorder>
   <zorder>critical_6</zorder>
   <zorder>critical_7</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>958</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
